homeassistant/components/openweathermap/config_flow.py

```diff
@@ -98,9 +109,13 @@ def __init__(self, config_entry: ConfigEntry) -> None:
         """Initialize options flow."""
         self.config_entry = config_entry
 
-    async def async_step_init(self, user_input=None):
+    async def async_step_init(self, user_input=None) -> ConfigFlowResult:
         """Manage the options."""
         if user_input is not None:
+            mode = user_input[CONF_MODE]
+            if mode == OWM_MODE_V30:
+                async_delete_issue(self.hass, self.config_entry.entry_id)
+
```

Comment:
Let's do this when the v30 entry is successfully setup

Reply:
ah ok, thanks.
didn't know it will work there

Reply:
done